packages/protocol/contracts/L1/libs/LibL1Tokenomics.sol

```diff
@@ -14,7 +14,7 @@ import {
 import {TaikoData} from "../TaikoData.sol";
 import {TaikoToken} from "../TaikoToken.sol";
 
-library LibTokenomics {
+library LibL1Tokenomics {
```

Comment:
Change name so we can also have a LibL2Tokenomics to calculate L2 EIP-1559 fees.